back/src/main/java/com/qmate/domain/questioninstance/service/AnswerService.java

```diff
@@ -50,15 +58,18 @@ public AnswerResponse create(Long questionInstanceId, Long userId, AnswerContent
     Answer saved = answerRepository.save(entity);
 
     // matchMember의 lastAnsweredAt 갱신
-    MatchMember matchMember = matchMemberRepository.findByMatch_IdAndUser_Id(qi.getMatch().getId(), userId).orElseThrow();
+    MatchMember matchMember = matchMemberRepository.findByMatch_IdAndUser_Id(qi.getMatch().getId(),
+        userId).orElseThrow();
     matchMember.updateLastAnsweredAt();
 
     // 완료 전이 (QI 행 잠금)
-    QuestionInstance locked = questionInstanceRepository.findByIdForUpdate(questionInstanceId).orElseThrow();
+    QuestionInstance locked = questionInstanceRepository.findByIdForUpdate(questionInstanceId)
+        .orElseThrow();
     if (locked.getStatus() == QuestionInstanceStatus.PENDING &&
         answerRepository.countDistinctUserIdByQuestionInstance_Id(questionInstanceId) >= 2L) {
       locked.markCompleted(LocalDateTime.now());
     }
+    petService.addExperienceForAnswerCompletion(locked.getMatch());
```

Comment:
```suggestion
        .orElseThrow();
    if (locked.getStatus() == QuestionInstanceStatus.PENDING &&
        answerRepository.countDistinctUserIdByQuestionInstance_Id(questionInstanceId) >= 2L) {
      locked.markCompleted(LocalDateTime.now());
      petService.addExperienceForAnswerCompletion(locked.getMatch());
    }
```
이부분 조건문 내부에 있어야 2명다 응답한게 되는거라 내부로 들어가면 될것같습니다

Reply:
내용 확인 후 해당내용 수정하였습니다. 확인 감사합니다. 수정 후 merge까지 진행하겠습니다.